client/python/armada_client/client.py

```diff
@@ -183,6 +183,47 @@ def get_queue_info(self, name: str):
         return response
 
 
+def search_for_job_complete(event, job_name: str, job_id: str) -> List[str]:
+    """Search the event stream to see if your job has finished running
+
+    :param event: a gRPC event stream
+    :param job_name: The name of your armada job
+    :param job_id: The name of the job id that armada assigns to it
+    :return: A list of [state, message]
+    """
+
+    tuple_return = ["temp", "temp2"]
```

Comment:
this is wrongish... a tuple in python is `('item1', 'item2')` (for N items)

Is there a reason you didn't want it to look like:
```
tuple_return = (
    "queued",
    f"Armada job:id {job_name}:{job_id} is queued"
)
```

Or alternatively, leave it a list (but don't pre-add items you don't need) and add the stuff as you do now, and rename the var.